pkg/chains/storage/gcs/gcs.go

```diff
@@ -122,7 +128,44 @@ type realGCSWriter struct {
 	bucket string
 }
 
+type gcsReader interface {
+	GetReader(object string) (io.ReadCloser, error)
+}
+
+type realGCSReader struct {
```

Comment:
i think we can just call this `reader`, since it's in the `gcs` package it'll imply that it's a gcs reader

Reply:
Done ✅ To be consistent, I also renamed `realGCSWriter` to `writer`.